notebooks/img/leafmap_mess.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #36.    #polygons = df_4001.apply(polygon, axis=1)
</pre><p>Should just delete these if they're not used anymore. If its something a user of the notebook would find useful by un-commenting, then that should be documented here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-PDS/search-api-notebook/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
resolved per https://github.com/NASA-PDS/search-api-notebook/pull/80/commits/01e5715ad91b5448c13758670f606620c8bf81bd